bridge/bridge/api/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,919 @@
+        explorerUrl:
+          description: URL of a block explorer for viewing transactions and accounts.
+          type: string
+          format: uri
+          example: https://explorer.testnet.example
+
+    WrapRequest:
+      type: object
+      description: Represents a detected wrap request originating on the native network.
+      required:
+        - requestTransactionHeight
+        - requestTransactionHash
+        - requestTransactionSubindex
+        - senderAddress
+        - requestAmount
+        - destinationAddress
+        - payoutStatus
+        - requestTimestamp
+      properties:
+        requestTransactionHeight:
+          description: |
+            Height of the block containing the request transaction.
+            Represented as a string to support very large block heights (string-encoded integer).
+          type: string
+          example: "12345678"
+        requestTransactionHash:
+          description: Hash of the transaction submitting the request.
+          type: string
+          example: 3f5b8f9a2bb2f6a9c3d2b4f0a1e2c3d4e5f6a7b8c9d0e1f2a3b4c5d6e7f8a90
+        requestTransactionSubindex:
+          description: Subindex for Symbol aggregate transactions or -1 for others.
+          type: integer
+          example: -1
+        senderAddress:
+          description: Address of the user initiating the request.
+          type: string
+          example: 0x12ab34cd56ef7890ab12cd34ef567890ab12cd34
+        requestAmount:
+          description: |
+            Amount of native tokens deposited for wrapping or unwrapping.
+            Amounts are returned as strings to preserve precision (decimal or integer string).
+          type: string
+          example: "1000000000000000000"
+        destinationAddress:
+          description: Address on the network that will receive the tokens after swap.
+          type: string
+          example: 0xE7DefF030632d08DA0ff3c5e9650459Dc8f746b7
+        payoutStatus:
+          description: Current payout processing status of this request (0 – unprocessed, 1 - sent, 2 - completed, 3 - failed).
+          type: integer
+          enum:
+            - 0
+            - 1
+            - 2
+            - 3
+          example: 0
+        payoutTransactionHash:
+          description: Hash of the payout transaction, if already initiated.
+          type: string
+          nullable: true
+          example: 03D848C2455CADBC26AB5F940D2232D168EFA6EA770D6F583E030C160769CB2E
+        requestTimestamp:
+          description: Timestamp (Unix) when the request transaction occurred.
+          type: integer
+          example: 1759781792
+        payoutTransactionHeight:
+          description: Block height of the payout transaction, if available (string-encoded integer).
+          type: string
+          nullable: true
+          example: "12345700"
+        payoutNetAmount:
+          description: Net payout amount sent after deducting fees (string-encoded integer).
```

Comment:
Must explain the units (whole, fractional, atomic...) and note that this is on the destination network, using the configured token.